configs/nginx/timenow.local

```diff
@@ -0,0 +1,23 @@
+server {
+	listen 443; 
+	listen [::]:443;
+
+	root /var/www/timenow; 
+
+	server_name timenow.local;
+
+	location /api/ { 
+		proxy_pass http://localhost:8080/;
+		add_header Access-Control-Allow-Origin http://timenow.local/api;
+	}
```

Comment:
When allowing cross-origin resource sharing (CORS), additional headers like Access-Control-Allow-Methods and Access-Control-Allow-Headers should also be added.